src/app/modules/item/components/item-header/item-header.component.ts

```diff
@@ -7,14 +10,38 @@ import { ItemData } from '../../services/item-datasource.service';
   templateUrl: './item-header.component.html',
   styleUrls: [ './item-header.component.scss' ]
 })
-export class ItemHeaderComponent {
+export class ItemHeaderComponent implements OnChanges {
   @Input() itemData?: ItemData;
 
+  navigationNeighbors: {parent: ItemRoute|null, left: ItemRoute|null, right: ItemRoute|null}|null = null;
```

Comment:
```suggestion
  navigationNeighbors: { parent: ItemRoute|null, left: ItemRoute|null, right: ItemRoute|null }|null = null;
```

Reply:
You have all nullable + the whole nullable. I don't think you interpret differently having it `null` or `{null, null, null}`. So either it has a different semantic, or it has the same and get rid of 2 ways to write the same.